__init__.py

```diff
@@ -98,17 +94,6 @@ def on_websettings_changed(self):
         # Otherwise new settings will not be regarded
         self._force_setup()
 
-    def __build_automation_intent(self):
-        intent = IntentBuilder("AutomationIntent").require(
-            "AutomationActionKeyword").require("Entity").build()
-        self.register_intent(intent, self.handle_automation_intent)
-
-    def __build_tracker_intent(self):
-        intent = IntentBuilder("TrackerIntent").require(
-            "DeviceTrackerKeyword").require("Entity").build()
-        # TODO - Identity location, proximity
-        self.register_intent(intent, self.handle_tracker_intent)
-
```

Comment:
Removing these will break the functionality for any language that hasn't added an `automation.intent` and `tracker.intent`. Could we leave the Adapt intents in with a deprecation warning to give the other languages time to do the translations?

This will also be a reminder to remove all the deprecated voc files eg `AutomationActionKeyword.voc`

Reply:
I wan to do cleanup in next PR.

For other languages, HA skill is already broken as other intents are already missing. For example, with SV-SE you cannot use even `turn on` as it already miss intens and contain only vocs for it.

I don't know what will be the lesser evil, if adding deprecate message and keeping a bloat in code in already broken language support or just pushing it and wait for users reaction and adding languages on the fly.
I once asked for help with translations for HA plugin to make this process synchronized at forum in past, but did not get  support for all languages.

Languages from active issuers here have support added, namely EN, CS, DE

What I miss are:
FR: `vocab/all_lights.voc` and `vocab/all_switches.voc`  @antipiot 
IT: `vocab/all_lights.voc`,  `vocab/all_switches.voc`, `vocab/tracker.intent` and automation.intent    @kamir86 or @forslund 

Could You please help with translations? Thank you guys in advance

Reply:
Hi Tony i'll check this out this week

Reply:
@antipiot, would You help with translation, please? Its only few words.

Reply:
hi tony, sorry for the late response, I made some PR on your branch with the required translations.
Let me know if I missed something

Reply:
Hi kamir, Thanks for IT translations, I had seen them when You sent them :) IT seems complete to me, so now I need only FR translations and it should be all. Other languages are already broken from previous commits so to me there is no point hold this PR any longer after FR is completed.

Reply:
> 
> 
> @antipiot, would You help with translation, please? Its only few words.

My apologises, not sure to understand as
 mycroft-homeassistant/vocab/fr-fr/all_switches.voc 
 and 
  mycroft-homeassistant/vocab/fr-fr/all_switches..voc 
Are already translated?

How can i help?

Reply:
@krisgesling @stratus-ss Hi, working localizations updated, commits squashed, PR should be ready.

Reply:
@krisgesling @stratus-ss Hi, any updates?

Reply:
Hey sorry, Stratus has been off for a bit. 

There's no easy way to do these big changes without breaking it for someone. So I'm going to approve this and hopefully we catch anything by people testing the default branch of the repo before it gets updated in the Marketplace.